packages/react-components/react-nav-preview/src/components/NavDrawerBody/NavDrawerBody.test.tsx

```diff
@@ -5,5 +5,6 @@ describe('NavDrawerBody', () => {
   isConformant({
     Component: NavDrawerBody,
     displayName: 'NavDrawerBody',
+    disabledTests: ['component-has-static-classnames-object'],
```

Comment:
For styling consistency, we might want to still create the navDrawerBodyClassNames and apply them respectively. I understand that it might be redundant, but we have created a contract where a slot will follow `fui-<component>__<slot>` so if someone wants to apply selectors for that specific slot they will have to look around and try to find what classname to use. this can be especially helpful for recomposing on the app side.

Reply:
ah - cool. Yeah, wilco - I drafted off ToolbarButton, that doesn't do this, but I like this idea better.